antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -925,9 +927,9 @@ def copy(
         shutil.copytree(src_path, dest_path)
 
         src_path = cast(Path, file_study.config.output_path)
-        if with_outputs and src_path.exists():
+        if with_outputs and src_path.exists() and output_ids:
```

Comment:
See other comment: we need to decide, if output_ids is empty, if this means that we want to copy all outputs.
I think it makes sense, since we can already copy no output at all with the 'with_outputs=False' option.